### PR TITLE
Exceedingly dirty fix to make sed in run script work properly

### PR DESCRIPTION
**Pull request by caseyfw**

### CODE REVIEW

**Review by singularo — APPROVED** [review]
So much dirt.
Comment, maybe.

**Review by singularo — APPROVED** [review]
Yep, I can't see any issues, merge so we can build and test.

File: s2i/bin/run [hunk 1]
```diff
@@ -24,9 +24,15 @@ if [ "${WEB_PATH}" == "/" ]; then
   echo "---> Unsetting \$WEB_PATH to avoid an apache alias being created."
   unset WEB_PATH
 else
-  sed -i "s@# RewriteBase /\$@RewriteBase ${WEB_PATH}@g" /code/web/.htaccess
+  # Set rewrite base in htaccess to WEB_PATH to generate correct URLs.
+  # Uses 'tacat' to output file to get around inability to create temporary files in this directory.
+  sed "s@# RewriteBase /\$@RewriteBase ${WEB_PATH}@g" /code/web/.htaccess \
+  | tac | tac | tee /code/web/.htaccess > /dev/null
```
Comment (bart88) [inline s2i/bin/run:30]: wow

Comment (caseyfw) [reply s2i/bin/run:30]: magic

**Pull request merged**